UnitsNet.Tests/UnitConverterTest.cs

```diff
@@ -35,6 +36,18 @@ public void ConvertByName_ConvertsTheValueToGivenUnit(double expectedValue, doub
             Assert.Equal(expectedValue, UnitConverter.ConvertByName(inputValue, quantityTypeName, fromUnit, toUnit));
         }
 
+        [Fact]
+        public void ConvertByName__QuantityCaseInsensitive()
+        {
+            Assert.Equal(0, UnitConverter.ConvertByName(0, "length", "Meter", "Centimeter"));
+        }
+
+        [Fact]
+        public void ConvertByName__UnitTypeCaseInsensitive()
```

Comment:
Is the double underscore intentional here and above?

Reply:
Unintentional.